services/NaturalLanguageUnderstanding/index.md

```diff
@@ -0,0 +1,100 @@
+---
+
+copyright:
+  years: 2017
+lastupdated: "2017-03-24"
+
+---
+
+{:new_window: target="_blank"}
+{:shortdesc: .shortdesc}
+{:screen: .screen}
+{:codeblock: .codeblock}
+{:pre: .pre}
+
+# Getting started with Natural Language Understanding
+{: #natural-language-understanding}
+
+With {{site.data.keyword.nlushort}}, developers can analyze semantic features of unstructured text to extract structured insights. The service returns information for categories, concepts, emotion, entities, keywords, metadata, relations, semantic roles, and sentiment.
+{:shortdesc}
+
+Follow these steps to get started with {{site.data.keyword.nlushort}}.
+
+## Create a service instance
+
+Go to the [Natural Language Understanding](https://bluemix.net/catalog/services/natural-language-understanding) page in the service catalog, log in to Bluemix, and click **Create**.
+
+To view your service credentials, go to your [list of services](https://console.{DomainName}/dashboard/services/) and click the service instance you created. From your service instance page, click the "Service Credentials" tab, then click **View Credentials** to view your username and password.
+
+## Make a request to the API
+
+To get started with one of the Watson Developer Cloud SDKs, view the SDK repositories for installation instructions and example usage:
+
+* [Node.js![External link icon](../../icons/launch-glyph.svg "External link icon")](https://github.com/watson-developer-cloud/node-sdk)
```

Comment:
We talked about this -- that I had missed the space between the link text and the icon. So all these external links should be like, `[Node.js ![External link icon]` (with the space before `!`)